### PR TITLE
Switch to the new Mono.Unix nuget

### PR DESCRIPTION
**Pull request by grendello**
The nuget replaces older `Mono.Posix-NETStandard` one and makes it
possible to build LibZipSharp using `dotnet` on Unix.

### CODE REVIEW

**Comment by grendello** [comment]
Hm, it seems native libraries aren't copied to the output directory from the Mono.Unix nuget

**Comment by grendello** [comment]
`dotnet build` did copy them, but not `msbuild`

File: LibZipSharp.props [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <_LibZipSharpNugetVersion>2.0.0</_LibZipSharpNugetVersion>
```
Comment (dellis1972) [inline LibZipSharp.props:3]: Do we need to bump the version in this PR?

Comment (grendello) [reply LibZipSharp.props:3]: Nope, 2.0.0 was never released, so it's fine

**Review by dellis1972 — APPROVED** [review]

**Pull request merged**